src/menus/config.py

```diff
@@ -11,4 +11,4 @@
     1: "Back",
     9: "Exit",
     0: "Home",
-}
```

Comment:
Just wondering if this symbol means that previously before the change there was no new line?

Reply:
The red circle with the line through it means that there is no newline and the end of the file, which is considered a formatting error by the PEP8 standard. It's not a big deal here however.